internal/commands/osmonitor/workflow.go

```diff
@@ -43,13 +44,11 @@ func RegisterWorkflows(e workflow.Engine) error {
 	return nil
 }
 
-func runReachabilityScan(
-	ctx context.Context,
-	ictx workflow.InvocationContext,
-) (uuid.UUID, error) {
-	cfg := ictx.GetConfiguration()
-	logger := ictx.GetEnhancedLogger()
-	errFactory := errors.NewErrorFactory(logger)
+func runReachabilityScan(cmdCtx context.Context) (uuid.UUID, error) {
```

Comment:
can we just call it `ctx`? i don't think there's any need to have a "command context". it's simply a context with a few dependencies on it, i think that's a very usual go pattern. imo no need to give it any special name.